cmd/krel/cmd/obs.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2023 The Kubernetes Authors.
```

can be done as follow-up, affects other files as well.